### PR TITLE
SQL Connector used for EKM using AKV is unsupported in Linux

### PR DESCRIPTION
**Pull request by inarocki**
The SQL Connector used for EKM using AKV is currently available for Windows alone. It is not available for SQL Server on Linux.  SQL Connector available at: https://www.microsoft.com/en-us/download/details.aspx?id=45344 However, the documentation says it applies to all versions. Request to update the documentation stating this feature is specific to SQL Server on Windows.

Added a note "Note: SQL Server TDE Extensible Key Management by using Azure Key Vault is currently unsupported for SQL Server on Linux."

### CODE REVIEW

File: docs/relational-databases/security/encryption/setup-steps-for-extensible-key-management-using-the-azure-key-vault.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ ms.author: arupp
 
 [!INCLUDE [SQL Server](../../../includes/applies-to-version/sqlserver.md)]
 
+**Note:** SQL Server TDE Extensible Key Management by using Azure Key Vault is currently unsupported for SQL Server on Linux.
```
Comment (amvin87) [inline docs/relational-databases/security/encryption/setup-steps-for-extensible-key-management-using-the-azure-key-vault.md:24]: We should probably re-phrase this as :
Extensible Key Management is not supported for SQL Server on Linux as documented here: https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-editions-and-components-2019?view=sql-server-ver16#Unsupported. Thus, this article does not apply for SQL Server on Linux.

**Review by amvin87 — CHANGES_REQUESTED** [review]

**Comment by amvin87** [comment]
Hi @rwestMSFT ! would you be available to help review the change and also help merge the requested change to this article.

**Comment by rwestMSFT** [comment]
@amvin87 Sure we can make the change. We have a content freeze right now so it will only happen later next week.

#reassign:rwestMSFT

**Review by Rupp29 — APPROVED** [review]
Looks like a reasonable update. Approved.

**Comment by Court72** [comment]
@Rupp29 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Rupp29** [comment]
#sign-off

**Comment by Court72** [comment]
Due to the upcoming Ignite conference, all merging and publishing is on hold until the morning of October 12.

This PR is ready for merge on 10/12.

**Comment by rwestMSFT** [comment]
This change will be processed in an internal PR as originally discussed.

#please-close

**Pull request closed without merging**